howdoi/howdoi.py

```diff
@@ -315,8 +375,11 @@ def command_line_runner():
         return
 
     if args['clear_cache']:
-        _clear_cache()
-        print('Cache cleared successfully')
+        if _clear_cache():
+            print('Cache cleared successfully')
+        else:
+            print('Error: clearing cache failed')
```

Comment:
Was thinking about the way we signal errors to the user. We now have `[ERROR]`, `Error:`, or simply the message itself like`'Sorry, couldn\'t find any help with that topic\n'`. I think I like simply saying that `Unable to clear the cache`.

Reply:
Yes, there should be one function that does that which has one format for standard messages and one for errors. It's not cache-related though and I wanted to avoid piling things in.

Reply:
With that said, it's an error and should be clearly indicated as such. If there are ever tests, they'll want it too.

Reply:
I agree that errors should be marked. Let's just pick one style `[ERROR]` or `Error:`

Reply:
ab18e0df0219c68f689731667e8a26966029414c